### ISSUE TITLE
The default hyperparameters don't produce the pre trained model if batchSize is 8.

### ISSUE THREAD

**Issue by kaleaht** [art-programmer/FloorplanTransformation#12]
Are the default hyperparameters (command line arguments) optimal for the model or is there some other hyperparameter set I should use to produce the papers results? The batch size is only 8 because of GPU limitations. Can the batch size be the problem?

**Comment by kaleaht** [comment]
Got the results. LR first with 0.1 until loss was about 3 and then with 0.01.

**Comment by hale2024** [comment]
Where you able to reproduce everything exactly
